pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -189,13 +189,8 @@ OFFSET_TO_PERIOD_FREQSTR: dict = {
     "WEEKDAY": "D",
     "EOM": "M",
     "BM": "M",
-    "BQS": "Q",
-    "QS": "Q",
```

Comment:
I find it odd that these "*start" offsets are here to begin with, as they don't have an associated Period:
```python
In [22]: MonthEnd()._period_dtype_code
Out[22]: 3000

In [23]: MonthBegin()._period_dtype_code
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[23], line 1
----> 1 MonthBegin()._period_dtype_code

AttributeError: 'pandas._libs.tslibs.offsets.MonthBegin' object has no attribute '_period_dtype_code'
```

So, trying this - let's see what fails

Reply:
nvm, this stuff is explicitly tested for...

https://github.com/pandas-dev/pandas/blob/a98be065e365c3d673d759414586345b14de50ac/pandas/tests/indexes/datetimes/methods/test_to_period.py#L53-L63